website/content/en/preview/getting-started/getting-started-with-terraform/_index.md

```diff
@@ -288,29 +256,36 @@ resource "helm_release" "karpenter" {
   namespace        = "karpenter"
   create_namespace = true
 
-  name       = "karpenter"
-  repository = "oci://public.ecr.aws/karpenter"
-  chart      = "karpenter"
-  version    = "{{< param "latest_release_version" >}}"
-
-  set {
-    name  = "serviceAccount.annotations.eks\\.amazonaws\\.com/role-arn"
-    value = module.karpenter_irsa.iam_role_arn
-  }
+  name                = "karpenter"
+  repository          = "oci://public.ecr.aws/karpenter"
+  repository_username = data.aws_ecrpublic_authorization_token.token.user_name
+  repository_password = data.aws_ecrpublic_authorization_token.token.password
+  chart               = "karpenter"
+  version             = "{{< param "latest_release_version" >}}"
 
   set {
     name  = "settings.aws.clusterName"
-    value = module.eks.cluster_id
```

Comment:
we will want to rely on `cluster_name` and not `cluster_id` since there is a new distinction coming with the support for EKS local clusters on ouposts https://github.com/hashicorp/terraform-provider-aws/issues/27560

Nothing required here other than where most instances of `cluster_id` are used, we should use `cluster_name`